client/src/components/pages/setting/components/information/index.tsx

```diff
@@ -2,18 +2,17 @@ import React from 'react';
 import clsx from 'clsx';
 import { Formik, Form, Field } from 'formik';
 import * as Yup from 'yup';
-import { FormInput } from 'components';
-import { Button } from 'components/basic';
+import { FormInput, Button } from 'components';
 import { ButtonClasses } from 'components/basic/button';
 import List from '../list-radio';
 import { IFormItem, IInformationProps } from './interfaces';
 import styles from './information.module.scss';
 
 const SignupSchema = Yup.object().shape({
-	name: Yup.string().min(2, 'Too Short!').max(50, 'Too Long!').required('Required'),
-	username: Yup.string().min(2, 'Too Short!').max(50, 'Too Long!').required('Required'),
-	clan: Yup.string().min(2, 'Too Short!').max(50, 'Too Long!').required('Required'),
-	skills: Yup.string().min(2, 'Too Short!').max(50, 'Too Long!').required('Required'),
+	name: Yup.string().min(2, 'Too short').max(50, 'Too long').required('Required'),
```

Comment:
Please, use here QA requirements for validation:
https://trello.com/c/xusLg7a2/44-sign-in-sign-up-notifications

or simply copy from here:
https://github.com/BinaryStudioAcademy/bsa-2021-codetrainer/blob/patch/username/client/src/components/pages/sign-up/index.tsx